pennylane_lightning/core/src/utils/cuda_utils/cuGates_host.hpp

```diff
@@ -653,17 +654,25 @@ template <class CFP_t, class U = double>
 static auto getSingleExcitation(U angle) -> std::vector<CFP_t> {
     const U p2 = angle / 2;
     const CFP_t c{std::cos(p2), 0};
-    const CFP_t s{std::sin(p2), 0};
+    // TODO: To remove conditional compilation here in the future, current
+    // implementation will block the simultaneous installation of LGPU and
+    // cutensornet backends
+
+#ifdef _ENABLE_PLGPU
+    const CFP_t s{-std::sin(p2), 0}; // column-major
+#else
+    const CFP_t s{std::sin(p2), 0}; // row-major
+#endif
```

Comment:
Thanks @AmintorDusko! Either `std::pow(-1, _ENABLE_PLGPU)` or `_ENABLE_PLGPU ?  :` wouldn't allow us to install both `PL-GPU` and `PL-Tensor` at the same time since `_ENABLE_PLGPU` is defined at compile-time with a single value. The easy fix could be creating a seprate file to store the info required info for `Lightning.Tensor`. Please let me know if you have any other solution in mind! Thanks

Reply:
Forgot to mention that the main risk of using `_ENABLE_PLGPU` is user could misuse this option and set it as any value other than `0` or `1`.

Reply:
This is a valid concern. But I consider that if a user sets it to everything, this user wants to enable it.
This is not a blocker, but we are working toward reducing the use of preprocessor directives on the Lightning repository.

Reply:
Ah, but if the number is odd, this will not work. I got you!